### PR TITLE
Add note about versioning in README

### LINKED ISSUE #7: Mention versioning in readme

### PR DESCRIPTION
**Pull request by robbevp**
Fixes #7 
I think we already agreed about versions following the API?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,4 +85,8 @@ new TracksScope.album(album_id)
 new TracksScope.artist(artist_id)
 new TracksScope.genre(genre_id)
 new TracksScope.filter(string) // Search in track titles
-```
\ No newline at end of file
+```
+
+## Versioning
+This package bases it's versioning on the major and minor versions of the [API](https://github.com/accentor/api). This way, changes in routes and scopes are introduced in the same version that they are introduced in the API.
```
Comment (chvp) [inline README.md:91]:
```suggestion
This package bases its versioning on the major and minor versions of the [API](https://github.com/accentor/api). This way, changes in routes and scopes are introduced in the same version that they are introduced in the API.
```

File: README.md [hunk 2]
```diff
@@ -85,4 +85,8 @@ new TracksScope.album(album_id)
...
+Patch releases can be issued to fix releases in this package itself.
```
Comment (chvp) [inline README.md:92]:
```suggestion
Patch releases can be issued to fix issues in this package itself.
```
I assume?

**Review by chvp — CHANGES_REQUESTED** [review]

**Comment by chvp** [comment]
I don't think we discussed this, but if you agree with what I said in #7 then that definitely seems like the best option to me.

**Comment by robbevp** [comment]
> I don't think we discussed this, but if you agree with what I said in #7 then that definitely seems like the best option to me.

Yeah, I clearly had the same idea

**Review by chvp — APPROVED** [review]

**Pull request merged**